sirepo/server.py

```diff
@@ -118,10 +118,12 @@ async def api_deleteSimulation(self):
         simulation_db.delete_simulation(req.type, req.id, qcall=self)
         return self.reply_ok()
 
-    @sirepo.quest.Spec(
-        "require_user", sid="SimId optional", filename="SimFileName", sim_data="SimData"
-    )
-    async def api_downloadFile(self, simulation_type, simulation_id, filename):
+    @sirepo.quest.Spec("require_user", filename="SimFileName")
+    async def api_downloadFile(self, simulation_type, filename):
```

Comment:
document that this is deprecated

Reply:
[f1bb3a2](https://github.com/radiasoft/sirepo/pull/6348/commits/f1bb3a233050a89f9eb071756adf7d55ee663fe1)
This fix is just a docstring; I looked at the `deprecation` package which is installed but I don't see used anywhere else. Is that something we want to start doing?